app/routes/login.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
```

Comment:
This file is redundant (the same route will be automatically generated by Ember).